tests/pw/SERIAL-COMMANDS.md

```
@@ -0,0 +1,402 @@
+```bash
+npm run check:pages             # See "trash" status
+npm run restore:pages           # Restore them
+npm run check:pages             # Verify "publish"
+npm run test:e2e                # Test
+```
+
+---
+
+## 📊 Command Dependencies Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)